dali/pipeline/util/worker_thread.h

```diff
@@ -89,10 +89,10 @@ class WorkerThread {
    * When the destructor is called other things that work() is using may have been gone long
    * before causing a hang. Now when Shutdown is called we are sure that all things around still exist.
    */
-  inline void Shutdown(void) {
+  inline void Shutdown(bool rethrow_worker_errors = true) {
```

Comment:
Do we have a legitimate case for rethrowing in shutdown? Isn't it ultimately always called from some kind of destructor?

Reply:
Yeah, one way or another it is always called from destructors.

Reply:
done